fantasy_football_app/models.py

```diff
@@ -10,7 +10,6 @@ class Player(models.Model):
         ('RB', 'Running Back'),
         ('WR', 'Wide Receiver'),
         ('TE', 'Tight End'),
-        ('K', 'Kicker'),
```

Comment:
Since you made a change to a model, you'll need to run makemigrations to create the migration file for this. Commit that with this branch too